### DISCUSSION TITLE
FluentSelect, multiple selctions, collapsed

### DISCUSSION THREAD

**Discussion by chrisonmoon** [microsoft/fluentui-blazor#2028] [Q&A]
Hello,

I'm new to Fluent Blazor, therefor my question might be not so clever. I have a problem: 

When i use FluentSelect with "multiple: false" the list is collapsed.
When i use FluentSelect with "multiple: true" the list is expanded.

Is there a way to have it with the following behaviour:
multiple: tru, and it always collapses after chabging the selection 

Thank you !

**Comment by vnbaaij** [comment]
Hi,

When you enable multiple selection, the select will always be shown expanded. We can't change that.

**Reply by chrisonmoon** [reply]
Dear Vincent - thanks a lot !!!!

**Comment by marciolordelo** [comment]
@chrisonmoon 

You have the FluentAutocomplete component as an option.
![image](https://github.com/microsoft/fluentui-blazor/assets/85967952/0059146f-8eb7-416b-8c04-a7de6dec5a7d)

![image](https://github.com/microsoft/fluentui-blazor/assets/85967952/5ad99e35-46f7-4bd0-af19-cb8d79a61574)

Or you can use a FluentButton (1) to display a FluentMenu (2), add FluentMenuItem elements with an event defined for the OnClick, and use the Checked property to control the selected elements.
![image](https://github.com/microsoft/fluentui-blazor/assets/85967952/f99b95f2-9231-4f71-a9e3-e4fbdf02ea40)

**Reply by chrisonmoon** [reply]
Hey Thanks so much !!!!

**Comment by TuckermanRav** [comment]
Remarkable, another thread on the same subject and this time the comment "We can't change that". As if making this UI control behave similarly to every other multi-select dropdown on the face of the earth is nothing short of landing a man on mars. All of the Fluent UI controls have been far more troublesome and cost too much engineering time ($$$) to do, what I would consider, trivial UI tasks. We have decided to pivot and take up a proper control library from the likes of DevExpress, Telerik and SyncFusion.

Thread from 2023: https://github.com/microsoft/fluentui-blazor/issues/792
